components/Account.tsx

```diff
@@ -1,55 +1,52 @@
 import { useState, useEffect } from "react";
-import PropTypes from "prop-types";
-import supabase from "../utils/supabaseClient";
+import useSupabase from "../hooks/useSupabase";
 
-export default function Account({ session }) {
+export default function Account() {
+  const { supabase, user } = useSupabase();
   const [loading, setLoading] = useState(true);
   const [username, setUsername] = useState(null);
   const [website, setWebsite] = useState(null);
-  // declared but not used for now fallback
-  const [, setAvatarUrl] = useState(null);
+  const [avatarUrl, setAvatarUrl] = useState(null);
 
-  async function getProfile() {
-    try {
-      setLoading(true);
-      const user = supabase.auth.user();
+  useEffect(() => {
+    const getProfile = async () => {
+      try {
+        setLoading(true);
 
-      const { data, error, status } = await supabase
-        .from("users")
-        .select("username, website, avatar_url")
-        .eq("id", user.id)
-        .single();
+        const { data, error, status } = await supabase
+          .from("users")
```

Comment:
The current Account tries to fetch info from `public.users` table, however we have our user info mapped into `public.profiles`. Changing `from` to `profiles` fixes the alert we get on incorrectly querying (checked locally)

Reply:
Fixed, thanks! (Sorry, I think you mentioned this earlier too, totally forgot)